<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCalibr</class>
 <widget class="QDialog" name="DialogCalibr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калибровка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_Value">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetCalPoints">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>9</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,10,10,10,10">
     <item>
      <widget class="QGroupBox" name="groupBoxChName">
       <property name="title">
        <string>Найменування канала</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEditChName"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Одиниці виміру канала</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_EU"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_averagePoints">
       <property name="title">
        <string>Усереднення</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_timeAvgInterval">
          <property name="text">
           <string>Час усередн., c</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_timeAvgInterval">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_avgPoints">
          <property name="text">
           <string>Кількість точок</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_numAvgPoints">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Калібровка</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Поточне</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Нове</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_CalibrVaueEU_new"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_CalibrValueRaw"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_EU">
          <property name="text">
           <string>EU</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_EU_new">
          <property name="text">
           <string>EU</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="buttonAddValue">
          <property name="text">
           <string>Додати значення</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/Add.ico</normaloff>:/icons/Add.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QPushButton" name="buttonDeleteValue">
          <property name="text">
           <string>Видалити значення</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/Delete.ico</normaloff>:/icons/Delete.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>rawUnits</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_CalibrValueEU"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Поточне</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Зміщення &quot;0&quot;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Ручн.вст.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_zeroOffset">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSetCurrentToOffset">
          <property name="text">
           <string>Встановити поточне як &quot;0&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonSaveCalibr">
       <property name="text">
        <string>Зберегти</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/Save.ico</normaloff>:/icons/Save.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="text">
        <string>Відміна/Вихід</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/exit.ico</normaloff>:/icons/exit.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_statistic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Statistic: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
